arcgis-ios-sdk-samples/Maps/Show location history/LocationHistoryViewController.swift

```diff
@@ -82,67 +122,85 @@ class LocationHistoryViewController: UIViewController {
     
     // MARK: Private behavior
     
+    private func makeDataSource(from polylineJSONString: String) -> AGSSimulatedLocationDataSource {
```

Comment:
FYI: Not a big deal since this is a private API, but from the [design guidelines](https://swift.org/documentation/api-design-guidelines/#strive-for-fluent-usage):

![image](https://user-images.githubusercontent.com/2257493/84587390-a8ce0480-add3-11ea-93ae-456bece68a7f.png)

So following the guidelines, this method should be named `makeDataSource(polylineJSONString: String)` and the following method should be named `makeOverlay(symbol: AGSSymbol)` or `makeOverlay(rendererSymbol: AGSSymbol)`.